meinberlin/apps/bplan/views.py

```diff
@@ -21,6 +23,12 @@ class BplanStatementFormView(ProjectContextDispatcher,
     template_name = 'meinberlin_bplan/statement_create_form.html'
     success_url = reverse_lazy('meinberlin_bplan:statement-sent')
 
+    def dispatch(self, request, *args, **kwargs):
+        project = self.get_project(*args, **kwargs)
+        if project.has_finished:
+            return HttpResponseRedirect(reverse('meinberlin_bplan:finished'))
```

Comment:
Why a redirect? I prefer showing the information text on the same URL.

Reply:
The `finished` view has different permissions from the statement submission view. With the redirect I was able to use the PermissionRequired mixin.